src/Core/ViewHelper/ViewHelperVariableContainer.php

```diff
@@ -29,6 +30,53 @@ class ViewHelperVariableContainer
      */
     protected $view;
 
+    /**
+     * Push a new delegate variable container to a stack. Done before you start
+     * sub-rendering with a ViewHelper that requires a special container for
+     * variables which is capable of being used in multidepth stacks without
+     * overriding the last instance or last set of variables.
```

Comment:
No matter how often I read this, I always fail to keep track of all the parts. ;-) Please split this in few, easier to understand, sentences.

Reply:
I will take that to heart and spend a bit more than 15 seconds documenting this method and the others ;)